src/workerd/io/actor-cache.c++

```diff
@@ -2581,7 +2582,10 @@ kj::Promise<void> ActorCache::flushImplUsingSinglePut(PutFlush putFlush) {
   // See the comment in flushImplUsingTxn for why we need to construct our RPC and then wait on
   // reads before actually sending the write. The same exact logic applies here.
   co_await waitForPastReads();
-  co_await request.send().ignoreResult();
+  {
+    util::DurationAlert alert(clock, kj::Duration(kj::SECONDS*1), "storage operation took longer than 1s: single put");
```

Comment:
Does this actually need the `kj::Duration(...)` around the calculation. Would `1 * kj::SECONDS` also yield a `kj::Duration`?

Reply:
That does indeed also work, thanks!